docs/dashboards/index.md

```diff
@@ -10,29 +12,29 @@ Log data is hard to analyze and can be extremely tedious to understand. Dashboar
 ## Creating Dashboards
 To create a Dashboard, start by going to the Dashboards screen and click the **+ Add New Dashboard** button. Give your Dashbord a name and click save.
 
-<video style="display:block; width:100%; height:auto;" controls autoplay>
-  <source src="https://cdn.sematext.com/videos/create-dash.mp4" type="video/mp4" />
-</video>
+![Create Dashboard](../images/dashboards/create-dashboard.gif)
+
+After creating your Dashboard, you'll notice that a default Overview Report is automatically generated in the left menu panel. Additionally, a dialog will appear on the screen, prompting you to create your first component using the [Chart Builder](https://sematext.com/docs/dashboards/chart-builder/) and add it to the report.
+
+### Chart Builder
 
-Follow along the [Quick Start](./quick-start) guide for more detailed instructions.
+<div class="video_container">
+<iframe src="https://www.youtube.com/embed/8PxtgrwY8SQ" 
+frameborder="0" allow="autoplay; encrypted-media" 
+allowfullscreen class="video"></iframe>
+</div>
+
+Learn more about [Chart Builder](https://sematext.com/docs/dashboards/chart-builder/).
 
 ## Dashboard Settings
-Dashboard settings include naming, editing, and cloning dashboards, and handling reports and components. You can choose to add any type of component to a dashboard, add, edit and clone reports and manage filters.
 
-<video style="display:block; width:100%; height:auto;" controls>
-  <source src="https://cdn.sematext.com/videos/dash-settings.mp4" type="video/mp4" />
-</video>
+On the Dashboards overview page, you have the ability to edit, delete, or clone your dashboards. Upon opening a dashboard, all the reports you've created within it will be visible in the left menu panel. Hovering over a report name and clicking on the three dots presents options to edit, clone, delete the report, and define [report variables](https://sematext.com/docs/dashboards/report-variables/).
```

Comment:
Note how here we talk about Dashboards, but then we also say "all the reports".  So maybe that's why we need to explain that there is a notion of top-level dashboards where one can pull in data from N solutions/apps, and App-scoped dashboards aka reports?

Note - this is where https://sematext.com/docs/images/dashboards/report-selector.png may be good to use because it shows what's under the three dots?

I do see a mention of report variables.... but that's under different three dots - the three dots in the top-right corner of dashboards and reports.

Reply:
The gif shows what's in ... on Dashboards page and what's in left menu panel once you open a dashboard. 
I also added:
Dashboards are top-level pages where you can add multiple reports in it.